### DISCUSSION TITLE
Neo-tree shows regular files as hidden

### DISCUSSION THREAD

**Discussion by ashermkn1** [LazyVim/LazyVim#428] [Q&A]
I am just using LazyVim right out of the box with no custom config or anything, and when I open neo-tree with <leader>-e, it says that all my files are hidden, here is a screenshot of it happening in the ~/.config/nvim directory but it happens in any other directory I try. Is this a bug with what neo-tree thinks are hidden files or am I just being dumb?
<img width="284" alt="image" src="https://user-images.githubusercontent.com/28579754/224866029-f39c7fbd-2578-405f-99f6-b7ad8256920a.png">
and here it is after toggling the hidden files with "H"
<img width="286" alt="image" src="https://user-images.githubusercontent.com/28579754/224866176-6913cae2-616b-430c-b3f0-0b1c9f6c081b.png">

**Comment by folke** [comment]
You probably have a misconfigured .gitignore or .fdignore file?

**Reply by ashermkn1** [reply]
I don't think I have any .gitignore files in the .config/nvim directory or any parent directories (just checked) and definitely don't have .fdignore files. Is there anything else it might be?

**Reply by folke** [reply]
no idea. check the neo-tree docs or ask it there. This is really not the place

**Reply by ashermkn1** [reply]
Ok, thank you for your help!
